tests/Kiota.Builder.Tests/Writers/Go/CodeMethodWriterTests.cs

```diff
@@ -635,7 +635,7 @@ public void WritesNullableVoidTypeForExecutor()
         };
         writer.Write(method);
         var result = tw.ToString();
-        Assert.Contains(" error {", result);
+        Assert.Contains("() {", result);
```

Comment:
I'm not sure this change is correct, the request executor by definition makes the request, so might encounter a whole range of different errors which need to be reported back to the user. Even if the expected response doesn't have content.